sycl/source/detail/scheduler/commands.cpp

```diff
@@ -681,15 +681,12 @@ static void adjustNDRangePerKernel(NDRDescT &NDR, RT::PiKernel Kernel,
 
   if (WGSize[0] == 0) {
```

Comment:
Is this condition valid to represent all "default work-group size" cases?

Reply:
WGSize[0] == 0 means user hasn't asked for a specific local size.